client/apis/volumegroupsnapshot/v1beta1/types.go

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 // +kubebuilder:object:generate=true
-package v1alpha1
+package v1beta1
 
```

Comment:
Do we need to drop `PVCVolumeSnapshotPair` from here? We are not using it by the looks of it.

Reply:
In my opinion, yes, and `PVVolumeSnapshotContentPair` is a leftover too.